### PR TITLE
update website event queue

### PR DESCRIPTION
**Pull request by judithpatudith**
## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

File: src/events.json [hunk 1]
```diff
@@ -424,18 +424,60 @@
...
+  { "title": "Containers, Persistent Data, Fast Data, Big Data (JavaOne)",
+      "image": "/assets/images/events/San_Francisco.jpg",
+      "date": "2017-10-05",
+      "url": "https://events.rainfocus.com/catalog/oracle/oow17/catalogjavaone17?search=container:%20check!%20But%20what%20about%20persistent&showEnrolled=false"
+  },
+
+  { "title": "SMACK Stack 2.0",
+      "image": "/assets/images/events/Copenhagen.jpg",
+      "date": "2017-10-18",
+      "url": "https://dockerconeurope.smarteventscloud.com/connect/search.ww#loadSearch-searchPhrase=smack&searchType=session&tc=0&sortBy=titleSort&p="
```
Comment (pleia2) [inline src/events.json]: https://dockerconeurope.smarteventscloud.com/connect/sessionDetail.ww?SESSION_ID=140455 is probably a better URL for this

File: src/events.json [hunk 2]
```diff
@@ -424,18 +424,60 @@
...
+  { "title": "Mesos Meetup Tokyo #3",
+      "image": "/assets/images/events/Tokyo.jpg",
+      "date": "2017-11-01",
+      "url": "https://mesos.connpass.com/event/67877/"
+  },
+
+  { "title": "The Open Sourcing of Infrastructure (CubaConf)",
+      "image": "/assets/images/events/Havana.jpg",
+      "date": "2017-11-09",
+      "url": "http://www.cubaconf.org/submissions.html"
```
Comment (pleia2) [inline src/events.json]: Let's just go with http://www.cubaconf.org/ for now, the submissions page is a bit weird (included everything submitted) so we'll just want to link directly to the session or schedule once it's online.

**Comment by pleia2** [comment]
retest this please

**Comment by judithpatudith** [comment]
Retest this please

**Comment by pleia2** [comment]
Built and content is fine, but we need to get to the bottom of this failing test.

**Comment by pleia2** [comment]
Retest this please

**Pull request merged**